src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/reporting/MoveFilterer.kt

```diff
@@ -55,5 +56,7 @@ object MoveFilterer {
 data class MoveFiltererParams(
         val supplier: Supplier,
         val movesFrom: LocalDate,
-        val movesTo: LocalDate,
-)
\ No newline at end of file
+        val movesTo: LocalDate) {
+
+    fun dateRange() = ClosedRangeLocalDate(movesFrom, movesTo)
```

Comment:
Added here to avoid changing the interface (constructor) and having rippled effect of changes elsewhere.